node/tests/RedisClient.test.ts

```diff
@@ -612,7 +615,7 @@ describe("GlideClient", () => {
             options.clientName = clientName;
             testsFailed += 1;
             client = await GlideClient.createClient(options);
-            return { client, context: { client } };
+            return { client, context: { client }, cluster };
```

Comment:
For my own knowledge, what is this returned object used for, and effect does adding "cluster" have?

Reply:
It allows to access `cluster` from within the test case in `SharedTests` https://github.com/valkey-io/valkey-glide/blob/11e9281d5f9b0fbe3c76147b856cfb023f3336b0/node/tests/SharedTests.ts#L82-L84